main.tf

```diff
@@ -11,6 +11,40 @@ resource "ibm_resource_instance" "scc_instance" {
   tags              = var.resource_tags
 }
 
+resource "ibm_scc_provider_type_instance" "scc_provider_type_instance_instance" {
+  depends_on       = [ibm_iam_authorization_policy.scc_wp_s2s_access]
+  count            = var.attach_wp ? 1 : 0
+  instance_id      = ibm_resource_instance.scc_instance.guid
+  attributes       = var.attributes
+  name             = var.provider_type_instance_name
+  provider_type_id = var.provider_type_id
+}
+
```

Comment:
can you a sleep in the code to workaround known issue: https://github.com/IBM-Cloud/terraform-provider-ibm/issues/4478

```
# workaround for https://github.com/IBM-Cloud/terraform-provider-ibm/issues/4478
resource "time_sleep" "wait_for_authorization_policy" {
  depends_on = [ibm_iam_authorization_policy.scc_wp_s2s_access]

  create_duration = "30s"
}
```